matplotcheck/tests/test_base.py

```diff
@@ -1,53 +1,113 @@
 """Tests for the base module"""
 import pytest
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotcheck.base import PlotTester
 
 
-def test_line_plot(pt_line_plt):
+@pytest.fixture
+def pd_df():
+    """Create a pandas dataframe for testing"""
+    return pd.DataFrame(
+        np.random.randint(0, 100, size=(100, 2)), columns=list("AB")
+    )
+
+
+@pytest.fixture
+def pd_scatter_plt():
```

Comment:
similarly @kysolvik  will you please have a look to ensure these fixtures aren't already in the conftest file?

Reply:
Hi @lwasser, just fixed this I think!